onnxruntime/python/tools/transformers/models/bart/utils/chain_enc_dec_with_beamsearch.py

```diff
@@ -0,0 +1,137 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.  See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+
+import onnx
+from onnx import TensorProto, helper
+from utils import export_helper
+
+
+def make_dim_proto_numeric(model, config):
+    """Make dim_proto numeric.
+
+    Args:
+        model (BartForConditionalGeneration): Bart model.
+        config: Bart config.
+    """
+    sequence_length = str(1)
+    num_heads = str(config.encoder_attention_heads)
+    hidden_size = str(config.d_model)
+    head_size = str(config.encoder_attention_heads)
+
+    for tensor in model.graph.output:
```

Comment:
please do this for model.graph.input as well

Reply:
Can I wait till #13130 been merged to address these. I would like to test it out.

Reply:
sure thing